components/FilteredRecipes.js

```diff
@@ -180,20 +181,20 @@ export default function FilteredRecipes({
           />
           <StyledSuggestionsList>
             {userInput &&
-              symptomSuggestions.map((suggestion, index) => (
+              symptomSuggestions.map((suggestion) => (
                 <StyledTextSuggestion
-                  key={index}
+                  key={uid()}
```

Comment:
this is unfortunately not better. With every render you call uid again, generating a new id. This means that the key changes every render. Please use a fixed value from your data array, for example "suggestion". If suggestion is not unique, please update the data structure to include predefined ids.

Please keep in mind that you need to update this everywhere you use uid this way.